tests/e2e/tslint.json

```diff
@@ -96,10 +96,10 @@
     "typedef": [
       true,
       // "call-signature",
-      "member-variable-declaration",
+      // "member-variable-declaration",
```

Comment:
It's reasonable to disable the rules that are preventing the image build to unblock the pipelines and fix the rules in a follow-up PR

Reply:
Got it.